includes/derivatives.inc

```diff
@@ -360,7 +360,6 @@ function islandora_ocr_derive_hocr(AbstractObject $object, $force = FALSE) {
     }
     // Clean up.
     file_unmanaged_delete($file);
-    islandora_ocr_remove_generate_ocr_rels_ext_statement($object);
```

Comment:
Was this supposed to be replaced with the "remove hocr flag" dealio? (and the "remove ocr flag" dealio moved to the "derive ocr" bit?)